CHANGES.rst

```diff
@@ -1,27 +1,27 @@
 Changes
 =======
 
-Version 0.8.0 (UNRELEASED)
+Version 0.8.0 (2021-11-22)
 --------------------------
 
 - Adds new command ``quota-show``.
```

Comment:
Please explain a bit, e.g. "to retrieve information about total CPU and Disk usage and quota limits".